src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignedAssemblyOrExcuse.java

```diff
@@ -131,7 +131,7 @@ public static void writeSAMFile( final String samFile, final PipelineOptions pOp
                     final List<BwaMemAlignment> alignments = allAlignments.get(contigIdx);
                     if ( alignments.isEmpty() ) continue;
                     final FermiLiteAssembly.Contig contig = assembly.getContig(contigIdx);
-                    final String readName = String.format("asm%06d:tig%05d", assemblyId, contigIdx);
+                    final String readName = formatAssemblyID(assemblyId) + ":" + formatContigID(contigIdx); //String.format("asm%06d:tig%05d", assemblyId, contigIdx);
```

Comment:
If we can be sure that the assembly and contig Ids are going be integers in nature in the foreseeable future, we can change the fields themselves in `AlignmentRegion` to integers. But the non-scientific part, i.e. the name of the reads produced in the (even when optional) SAM file and the annotations in the final VCF must have an agreement or at least have a protocol to understand each other.